website/src/css/imports/docs.css

```diff
@@ -21,7 +21,7 @@
     @media screen and (min-width: $desktop) {
       position: fixed;
```

Comment:
I believe @verythorough is talking would mean changing this line from `fixed` to `absolute`, so that the menu stays at the top, instead of scrolling with the content. You may want to `float` it or something else, but I believe that's the main idea.

Reply:
I actually meant removing the media query altogether, but I accidentally said 768px instead of 960px (which is what `$desktop` represents). If you remove this, you also need to remove the other $desktop media query, on line 120.